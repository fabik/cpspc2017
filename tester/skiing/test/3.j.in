1
98920199 939313512 -198762785 -4953616
-584657728 -124342039 231113166
