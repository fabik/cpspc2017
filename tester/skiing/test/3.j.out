2
98920199 939313512
-198762785 -4953616
